### DISCUSSION TITLE
Facing issue while running Rustdesk client on windows 32 bit macine

### DISCUSSION THREAD

**Discussion by Chapra78** [rustdesk/rustdesk#8586] [Q&A]
Hello guys,
We have deployed a self hosted Ruskdesk server (192.168.1.10) in our LAN.
Then we have build the Rustdesk Client using the sourcecode downloaded from "https://github.com/rustdesk/rustdesk" and by following the steps mentioned at https://github.com/infiniteremote/installer/files/14814894/Setup_rustdesk_development_on_Windows.pdf
We have hardcoded our server IP (192.168.1.10) and Rustdesk server key in the source code.
The client is working successfully on all our 64 bit windows machines.Unfortunately we are having issues running the Rustdesk client on 32 bit windows machines. Hence please let us know how to resolve this issue and how to build the client for running on 32 bit Windows machines.

**Comment by diocesangsmith** [comment]
I second this request. We would love to switch to RustDesk but the lack of a client that will run on a 32 bit machine is a deal breaker.
